_startups/ban.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Base Adresse Nationale
+mission: "Un référentiel national ouvert : de l'adresse à la coordonnée géographique."
+domain: adresse.data.gouv.fr
+repository: https://github.com/etalab/ban
```

Comment:
Depends who is targeted: https://github.com/etalab/ban is where the work currently focuses (CRUD API to make it simple), and that *will* be the actual **B** of BAN,  but it's not yet in production. So if we want to say "here is where the team is working", that's OK. If we want to say "here is the code actually released", better to use https://github.com/etalab/adresse.data.gouv.fr as an entry point I'd say.